app/app/schemas/users/student.py

```diff
@@ -0,0 +1,50 @@
+from typing import Optional
+
+from pydantic import BaseModel, validator
+
+from ..term import Term
+from .user import User
+
+
+# Shared properties
+class StudentBase(BaseModel):
+    user_id: Optional[int] = None
+    term_id: Optional[int]
```

Comment:
no default value?

Reply:
Ah, had added it locally later, forgot to commit